vtds_cluster_kvm/private/cluster.py

```diff
@@ -535,22 +561,24 @@ def __set_all_connected_blade_macs(
             self.__set_connected_blade_macs(network, prefix)
 
```

Comment:
REVIEWERS: in order for things to work better, I have moved the preparation of the configuration (inheritance and any massaging needed for deployment) out of the `prepare` phase of setting up the vTDS system and into the `consolidate` phase which is intended for configuration massaging. This is also where the connected blade caching call is made.